core/templates/dev/head/pages/creator_dashboard/CreatorDashboard.js

```diff
@@ -416,29 +416,32 @@ oppia.controller('CreatorDashboard', [
                 reviewMessage: $scope.reviewMessage
               });
             };
-            $scope.editSuggestion = function(){
-              $scope.showSuggestionEditor = true;
+            $scope.editSuggestion = function() {
+              $scope.suggestionEditorIsShown = true;
             };
             $scope.cancelReview = function() {
               $uibModalInstance.dismiss();
             };
-            $scope.showEditButton = function(){
-              return (!$scope.isNotHandled && $scope.isSuggestionRejected &&
-                !$scope.showSuggestionEditor);
+            $scope.isEditButtonShown = function() {
+              return (
+                !$scope.isNotHandled && $scope.isSuggestionRejected &&
```

Comment:
For multiline expressions in parentheses, break after "(".

/cc @apb7 I thought this was fixed? It's been a long-standing issue.

Reply:
@seanlip, I looked into this --  this hasn't yet been fixed, we fixed this problem for python files. I have opened issue #6047 for this. Thanks!